contracts/masset/liquidator/Unliquidator.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.8.6;
+
+import { IClaimRewards } from "../../polygon/IClaimRewards.sol";
+import { ImmutableModule } from "../../shared/ImmutableModule.sol";
+
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/**
+ * @title   Unliquidator
+ * @author  mStable
+ * @notice  Replacement Contract for the Liquidator.
+ *          Does not liquidate the tokens but sends them to the treasury or a set address
+ * @dev     VERSION: 1
+ *          DATE:    2022-02-10
+ */
+contract Unliquidator is ImmutableModule {
+    using SafeERC20 for IERC20;
+
+    event ClaimedAndSendRewards(address from, address token, uint256 amount, address to);
+    event NewReceiver(address receiver);
+
+    /// @notice Address to which the tokens will be send at the end
+    address public receiverSafe;
+
+    constructor(address _nexus, address _receiverSafe) ImmutableModule(_nexus) {
+        require(_receiverSafe != address(0), "Invalid receiver address");
+        receiverSafe = _receiverSafe;
+    }
+
+    /***************************************
+                    GOVERNANCE
+    ****************************************/
+
+    /**
+     * @notice Sets a new receive address for the tokens
+     * @param  _receiver  Address to which the tokens will be send at the end
+     */
+
+    function setReceiver(address _receiver) external onlyGovernance {
+        //
+        require(_receiver != address(0), "Invalid receiver address");
+        receiverSafe = _receiver;
```

Comment:
Usually the same name is kept in order to have consistency . 
        receiverSafe = _receiver;